src/Commands/Commands.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace Drupal\os2forms_organisation\Commands;
+
+use Drupal\os2forms_organisation\Helper\OrganisationHelper;
+use Drush\Commands\DrushCommands;
+use ItkDev\Serviceplatformen\Service\Exception\SoapException;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * A drush command file for commands related to os2forms_organisation.
+ */
+class Commands extends DrushCommands {
+
+  /**
+   * The organisation helper.
+   *
+   * @var \Drupal\os2forms_organisation\Helper\OrganisationHelper
+   */
+  private OrganisationHelper $helper;
+
+  /**
+   * The read options.
+   *
+   * @var array
+   * @phpstan-var array<string, mixed>
+   */
+  private array $readOptions = [];
+
+  /**
+   * Constructor.
+   */
+  public function __construct(OrganisationHelper $helper) {
+    parent::__construct();
+    $this->helper = $helper;
+  }
+
+  /**
+   * Lookup provided bruger id.
+   *
+   * @param string $brugerId
+   *   string Argument with organisation bruger id.
+   *
+   * @command os2forms_organisation:lookup
+   *
+   * @usage os2forms_organisation:lookup
+   */
+  public function lookup(string $brugerId): void {
+    $this->output()->writeln('Name: ' . $this->helper->getPersonName($brugerId));
+  }
+
+  /**
+   * Read data.
+   *
+   * @param string $type
+   *   The object type to read (bruger).
+   * @param string $uuid
+   *   The UUID.
+   * @param array $options
+   *   The options.
+   *
+   * @option manager-levels
+   *   Levels of managers to read
+   *
+   * @command os2forms_organisation:read
+   *
+   * @usage os2forms_organisation:read --help
+   *
+   * @phpstan-param array<string, mixed> $options
+   */
+  public function read(string $type, string $uuid, array $options = [
+    'manager-levels' => 1,
+  ]): void {
+    $this->readOptions = $options;
+
+    switch ($type) {
+      case 'bruger':
+        $data = $this->readBruger($uuid);
+
+        $this->output()->writeln(Yaml::dump($data, PHP_INT_MAX));
+        break;
+
+      default:
+        throw new \InvalidArgumentException(sprintf('Unknown type: %s', $type));
+    }
+  }
+
+  /**
+   * Read person.
```

Comment:
Should this say `Read bruger` to avoid confusion?

Reply:
Yes. I'll fix it.